Bottom: 84a295eebf6ed576a13cf5de2813ba8d0c86a68d
Top:    b3822f07ed276a52b5c5bc866eaba06753f41f41
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 20:31:10 +0100

Refresh of add-a-hacky-name-mangling-thin

---

diff --git a/src/file.c b/src/file.c
index e8f4d98..35f00c6 100644
--- a/src/file.c
+++ b/src/file.c
@@ -353,11 +353,11 @@ LoadPCB (char *Filename)
   clock_t start, end;
   double elapsed;
 
-  start = clock();
-
   /* Hack, load the mapping file */
   RefdesMapInit ();
 
+  start = clock();
+
   /* new data isn't added to the undo list */
   if (!ParsePCB (newPCB, Filename))
     {
